### PR TITLE
custom gapps

### PR DESCRIPTION
**Pull request by owengque**

### CODE REVIEW

File: owengq.config [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+GoogleMessages=1
+GoogleDialer=1
+GoogleContacts=1
+CarrierServices=0
+GoogleClock=1
+
+# Set SetupWizard=0 if you want to skip installing all packages belonging to SetupWizard Package
+SetupWizard=1
+>>SetupWizard=1
+>>GoogleRestore=0
```
Comment (nikhilmenghani) [inline owengq.config:78]: GoogleRestore should not be 0, SetupWizard comes as a whole, I would suggest you to change GoogleRestore=1 and send the pull request again

File: owengq.config [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+GoogleRecorder=0
+GoogleTTS=0
+GoogleTalkback=0
+DeviceSetup=0
+AndroidAuto=0
+
+# Set GoogleChrome=0 if you want to skip installing all packages belonging to GoogleChrome Package
+GoogleChrome=1
+>>GoogleChrome=0
+>>WebViewGoogle=1
```
Comment (nikhilmenghani) [inline owengq.config:133]: WebView and Trichromelibrary without GoogleChrome is not useful. Please change GoogleChrome=1 here and send the pull request again

**Review by nikhilmenghani — COMMENTED** [review]
Please go through the comments, make the changes and send the pull request again

**Comment by nikhilmenghani** [comment]
@owengque if there is no update on this today, I will close the pull request.

**Comment by nikgapps** [comment]
Closing because of no response

**Pull request closed without merging**